cves/kernel/CVE-2021-38166.yml.bak

```diff
@@ -147,7 +146,7 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
+  answer: I followed the links on the CVE page, but it seems like in the same e-mail the developer sent reporting the vulenrability, he also described how to fix it. This e-mail is the main source of information about the vulnerability.
```

Comment:
You might want to include links to other places you looked (other resources) even if they aren't the primary source for information on this vulnerability.